ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -105,11 +105,32 @@ def _extract_input_message(self, messages, system_prompt=None):
 
             elif isinstance(content, list):
                 for block in content:
-                    if block.get("type") == "text":
-                        input_messages.append({"content": block.get("text", ""), "role": role})
-                    elif block.get("type") == "image":
+                    if _get_attr(block, "type", None) == "text":
+                        input_messages.append({"content": _get_attr(block, "text", ""), "role": role})
+
+                    elif _get_attr(block, "type", None) == "image":
                         # Store a placeholder for potentially enormous binary image data.
                         input_messages.append({"content": "([IMAGE DETECTED])", "role": role})
+
+                    elif _get_attr(block, "type", None) == "tool_use":
+                        name = _get_attr(block, "name", "")
+                        inputs = _get_attr(block, "input", "")
+                        input_messages.append(
+                            {"content": "\n\n[tool: {}]\n\n{}".format(name, json.dumps(inputs)), "role": role}
```

Comment:
```suggestion
                            {"content": "[tool: {}]\n\n{}".format(name, json.dumps(inputs)), "role": role}
```
We do not need to inject leading newlines

Reply:
done